package.json

```diff
@@ -17,19 +17,19 @@
   "dependencies": {
     "iter-tools": "^1.4.1",
     "lodash": "^4.17.11",
-    "micromatch": "^3.1.4",
+    "micromatch": "^4.0.4",
     "obj-delta": "0.0.3",
     "obj-path-expression-parser": "0.0.2"
```

Comment:
i think this should point to the Tes repo now, and not Maurizio's
eg:
"obj-path-expression-parser": "github:tes/obj-path-expression-parser"